core/src/main/kotlin/entity/interaction/ContextInteraction.kt

```diff
@@ -51,10 +65,7 @@ class GlobalChatInputCommandInteraction(
  */
 sealed interface  UserCommandInteraction : ApplicationCommandInteraction {
 
-    val name: String get() = data.data.name.value!!
-
-    private val resolvedUsersData  get() = data.data.resolvedObjectsData.value?.users?.value
-    val users get() = resolvedUsersData.orEmpty().mapValues { User(it.value, kord) }
+    val users get() = resolvedObjects!!.users
```

Comment:
This seems to have changed the nullability of `users` - before this change it was non-null, now it's nullable because that's how it is on `ResolvedObjects`

I'm not sure it can contain more than one value for a `UserCommandInteraction` anyway?